TicTacToe/Logic/HumanPlayer.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace TicTacToe.Logic
+{
+    public class HumanPlayer : Player
+    {
+        public HumanPlayer()
+        {
+            _idPlayer = PlayerMarker.X;
```

Comment:
This constructor can leverage the `:this()` syntax.

Please google C# constructor invoke another constructor

Reply:
I apply this comment also in this class - "This code looks like you set data members of base class - this goes again C# standards.
The data members of base class should be private, and you should pass their values to base constructor using: base(...) syntax"